test-utils/runtime/src/lib.rs

```diff
@@ -666,10 +666,8 @@ fn code_using_trie() -> u64 {
 	if let Ok(trie) = TrieDB::<Hashing>::new(&mdb, &root) {
 		if let Ok(iter) = trie.iter() {
 			let mut iter_pairs = Vec::new();
-			for pair in iter {
-				if let Ok((key, value)) = pair {
-					iter_pairs.push((key, value.to_vec()));
-				}
+			for (key, value) in iter.flatten() {
+				iter_pairs.push((key, value.to_vec()));
 			}
 			iter_pairs.len() as u64
```

Comment:
This whole thing is just an `iter.flatten().count() as u64`

Reply:
Good spot!